_posts/2021-02-25-

```diff
@@ -0,0 +1,55 @@
+---
+title:  "Notes: February 2021 Foundation for Public Code community call"
+date: 2021-02-25
+author: elena, ryan 
+type: blogpost
+excerpt: Introducing the publiccode.net product requirements document
+category: community call
+---
+
+
+# Notes: February 2021 Foundation for Public Code community call 
+
+In this community call, we talked through our thinking about [publiccode.net](https://publiccode.net/) as product ([pull request #169](https://github.com/publiccodenet/publiccode.net/pull/169)), and demo'd [our wireframes](https://www.figma.com/file/Mcz4evZVI4rRwv3quXtKGC/Site-Map-Wireframe-exploration-Foundation-for-Public-Code?node-id=177%3A0).
+
+
+## Attendees
+
+Alba Roza
```

Comment:
Was this intended to be an unordered list?

Reply:
Fixed!